recipes/CommonVoice/ASR/CTC/hparams/train_de_with_wav2vec.yaml

```diff
@@ -0,0 +1,177 @@
+# Model: wav2vec2 + DNN + CTC
+# Augmentation: SpecAugment
+# Authors: Titouan Parcollet 2021
+#          Mirco Ravanelli 2021
+#          Sangeet Sagar 2022 (minor edits)
+# Language: German (DE)
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 8200
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/train_wav2vec2_char/<seed>
```

Comment:
We should have the same nomenclature for every language recipe in SpeechBrain, as such, you should use "wav2vec_ctc_de" instead.

Reply:
Fixed.